newIDE/app/src/SceneEditor/Toolbar.js

```diff
@@ -107,6 +102,12 @@ export class Toolbar extends PureComponent {
           buildMenuTemplate={() => [
             {
               type: 'checkbox',
+              label: 'Show Mask',
+              checked: this.props.isWindowMaskShown(),
+              click: () => this.props.toggleWindowMask(),
+            },
+			{
```

Comment:
This is mis-indented ;) To have automatic indentation and avoid losing time on this, the whole codebase is automatically formatted by Prettier. You can install the Prettier extension for your text editor: see https://github.com/4ian/GD/blob/master/newIDE/README.md#recommended-tools-for-development

Reply:
(btw, this is surely because your editor is configured to show tabs as 4 spaces. Using an editor like Visual Studio Code, the detection of the indentation is automatic. The whole codebase of GDevelop is based on 2 spaces for indentation as it's the most commonly used formatting 😃)